### PR TITLE
correcting the wrong Class name to right Class Name along with creating the convert_tags_to_color function

### PR DESCRIPTION
**Pull request by achiverram28**

### CODE REVIEW

**Comment by achiverram28** [comment]
I have added the required function called as convert_tags_to_color in printer.py , and made some necessary required changes , please check the pr and give it a hacktober-fest accepted tag

File: README.md [hunk 1]
```diff
@@ -43,11 +43,15 @@ printer.configure_options(PrinterConfig(delimiter_left="|", delimiter_right="|",
...
+from fancyprint.helpers.constants import Align
+printer=Printer()
+printer.print("This is some text in the center", PrintConfig(align=Align.CENTER, blank_character=" ", left_delimiter=True, right_delimiter=True, back_separator=True,
               front_separator=True))
 
 # Example
-printer.print("Hello World!", PrinterConfig(align=Align.CENTER, blank_character=" ", left_delimiter=True))
+printer.print("Hello World!", PrintConfig(align=Align.CENTER, blank_character=" ", left_delimiter=True))
 
 ```
```
Comment (IamMU) [inline README.md:56]: Nice catch :+1:

File: fancyprint/printer.py [hunk 2]
```diff
@@ -229,6 +219,23 @@ def print(self, text: str, print_config = PrintConfig()):
 
         if print_config.front_separator:
             self.separate_line("front")
+    def log(self, message: str, log_config = LogConfig()):
```
Comment (IamMU) [inline fancyprint/printer.py:225]: Add 2 spaces between function end and new function

**Review by IamMU — CHANGES_REQUESTED** [review]
Epic job @achiverram28 :fire: 
See the comments/with the requested change(very minimal - have code_of_conduct on the works currently :wink: )

**Comment by achiverram28** [comment]
I have added the spaces

**Review by IamMU — COMMENTED** [review]
Epic work @achiverram28 :+1:

**Comment by IamMU** [comment]
@achiverram28 you changed the file structure :|
Make another pr with the correct structure, i am marking this as invalid

**Comment by achiverram28** [comment]
I had changed a lot of files, and cannot make out what I have made, and src was not required , it was extra , and please comment what changes to make , but after working a lot on it and getting an invalid tag is not worthy, please if possible remove it, it's a request,  it will affect my hacktoberfest .Please I guess you understand

**Comment by IamMU** [comment]
@achiverram28 you're right, the pr doesn't deserve a invalid tag. Hacktoberfest doesn't have good rules and needs more explaining for maintainers. Don't worry, it will be removed once you add the src folder back. The project follow the latest standards(https://packaging.python.org/en/latest/tutorials/packaging-projects/) and the new structure. Please consider following the standard and making a new pr for that. :slightly_smiling_face: 
Don't worry about this pr being invalid, make another one and I'll accept it. You'll get 2 points then i guess. I'll have to ask in the discord server to be sure.
Please do the requested changes.

**Pull request merged**